test/integration/util/util.go

```diff
@@ -468,17 +495,23 @@ func UpdateNodeStatus(cs clientset.Interface, node *v1.Node) error {
 func InitTestAPIServer(t *testing.T, nsPrefix string, admission admission.Interface) *TestContext {
 	_, ctx := ktesting.NewTestContext(t)
 	ctx, cancel := context.WithCancel(ctx)
-	testCtx := TestContext{Ctx: ctx}
+	testCtx := &TestContext{Ctx: ctx}
```

Comment:
My goal was to not change the prototype of `InitTestAPIServer`.

Therefore the wrapper can be set after the `TestContext` and all servers have been created.